recipes/boltons/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "16.3.0" %}
+
+package:
+    name: boltons
+    version: '{{ version }}'
+
+source:
+    git_url: https://github.com/mahmoud/boltons.git
+    git_rev: '{{ version }}'
```

Comment:
Can you change this to:

```yaml
fn: boltons-{{ version }}.tar.gz
url: https://github.com/mahmoud/boltons/releases/download/{{ version }}/boltons-{{ version }}.tar.gz
sha256: be9aa2b6ffc74b526b1fe080272fd5706e05747f40166e3ca60ea04e34ca8816
```

We try to avoid cloning the whole repository, and the checksum is to ensure we have the right file.